Gopkg.toml

```diff
@@ -23,7 +23,13 @@
 #   non-go = false
 #   go-tests = true
 #   unused-packages = true
-
+required = [
+  "k8s.io/code-generator/cmd/client-gen",
```

Comment:
kept them to use in next version for generating client , informer , lister.

Reply:
They will not increase the binary size of edge_core as well as edgecontroller. So its fine :)